recipe/meta.yaml

```diff
@@ -29,33 +29,38 @@ outputs:
       run_exports:
         - {{ pin_subpackage('openmpi', min_pin='x.x.x', max_pin='x') }}
       ignore_run_exports:
-        - ucx  # [enable_cuda]
+        - ucx  # [linux]
       script_env:
-        - CUDA_HOME     # [enable_cuda]
+        - CUDA_HOME  # [enable_cuda]
     requirements:
       build:
         - {{ compiler('c') }}
         - {{ compiler('cxx') }}
         - {{ compiler('fortran') }}
-        - autoconf  # [unix]
-        - automake  # [unix]
+        #- autoconf  # [unix]
+        #- automake  # [unix]
         - libtool   # [unix]
         - make  # [unix]
-        - perl 5.26.2
       host:
+        #- openpmix
+        #- prrte
+        - libhwloc
+        - libevent
         - zlib
-        - ucx                 # [enable_cuda]
-        - ucx-proc =*=gpu     # [enable_cuda]
+        - ucx  # [linux]
       run:
-        - zlib
         - mpi 1.0 openmpi
+        #- openpmix
+        #- prrte
+        - libhwloc
+        - libevent
+        - zlib
       run_constrained:
```

Comment:
Missed `ucx`:
```suggestion
      run_constrained:
        - {{ pin_compatible('ucx', max_pin='x.x') }}  # [linux]
```

Reply:
No, I didn't miss it, look below. However, we had maxpin=x, should we maxpin=x.x instead as in your suggestion?

Reply:
Hmmm you're right, sorry I blame my naughty browser. I think `x.x` is safer, it's what we do in its `run_exports` too: https://github.com/conda-forge/ucx-split-feedstock/blob/49ca22ea4107626d829bbf31973ff117054cdd40/recipe/meta.yaml#L22